shipyard_blueprints/snowflake/shipyard_snowflake/cli/upload.py

```diff
@@ -119,75 +118,79 @@ def main():
             )
             sys.exit(snowflake_client.EXIT_CODE_INVALID_CREDENTIALS)
 
+        snowflake_client.connect()
+
         if args.snowflake_data_types != "":
```

Comment:
personally here I would've simplified this to make it a bit more cleaner
`if args.snowflake_data_types:`